compiler/testData/codegen/bytecodeText/sam/samWrapperInInlineLambda.kt

```diff
@@ -16,4 +16,5 @@ fun box(): String {
     return result
 }
 
-// 1 INVOKESPECIAL TestKt\$sam\$i\$java_lang_Runnable\$0.<init> \(Lkotlin/jvm/functions/Function0;\)V
\ No newline at end of file
+// 1 INVOKESPECIAL TestKt\$sam\$java_lang_Runnable\$0.<init> \(Lkotlin/jvm/functions/Function0;\)V
```

Comment:
@sfs Could you also same test in case when box is inline

Reply:
Done (`compiler/testData/codegen/bytecodeText/sam/samWrapperInInlineLambda2.kt`)